packages/block-editor/src/components/block-switcher/style.scss

```diff
@@ -77,7 +78,6 @@
 		margin: 0 auto;
 		display: flex;
 		align-items: center;
-		min-width: 100%;
```

Comment:
I'm not sure what side effect this change might have. Alternative options could be the following method:

```css
.components-button.block-editor-block-switcher__no-switcher-icon .block-editor-blocks-icon {
    width: 24px !important;
}
```

Reply:
I decided to go with this method :)